cql3/Cql.g

```diff
@@ -1419,7 +1419,7 @@ serviceLevelOrRoleName returns [sstring name]
 						 std::transform($name.begin(), $name.end(), $name.begin(), ::tolower); }
 | t=STRING_LITERAL     { $name = sstring($t.text); }
 | t=QUOTED_NAME        { $name = sstring($t.text); }
-| k=unreserved_keyword { $name = sstring($t.text); 
+| k=unreserved_keyword { $name = k;
```

Comment:
please add a unit test that reproduces the crash without the fix.

Reply:
or a link to the dtest

Reply:
I have open a porting request for the sla dtests.
https://github.com/scylladb/scylla-dtest/issues/2902

Reply:
Note, a unit test is better here since it catches the problem earlier and doesn't go through the motions of setting up a cluster. Keep in mind for similar cases.